### DISCUSSION TITLE
multipar diving into subdirectories

### DISCUSSION THREAD

**Discussion by magnemann** [Yutaka-Sawada/MultiPar#51] [Q&A]
Hi,
Im making a cold storage of 300GB and will try to make lots of PAR2 files.
Multipar uses my gpu, reducing time for making par2 for 10GB of data from 41 min to 7 min. Awesome!

I wonder, is there å way to get multipar to select subdirectories recursively? I have many directories at many levels.

Thank for a great project!

**Comment by Yutaka-Sawada** [comment]
There is an option for recursive search. Open "MultiPar Options" window, select "Client behavior" tab, there is an item of "Don't search subfolders" in "Creation options" section. When you check the option, it doesn't search files in sub-directories recursively. If you want recursive search, remove the check.

**Reply by magnemann** [reply]
Thanks!
It worked like a charm. Right-click a folder->multipar->create recovery files.
I will also encrypt the backup with gocryptfs. (the successor of encfs.)
But I am unsure if I shall make PAR files of the encrypted files or the unencrypted files?

best regards, Magne

**Reply by Yutaka-Sawada** [reply]
When you want to encrypt source files, you must encrypt them at first. Then, create recovery files for the encrypted files.
